build.gradle

```
@@ -76,6 +77,9 @@ dependencies {
     testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.3.1'
     testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: '5.3.1'
     testRuntime group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.3.1'
+    compileOnly 'systems.manifold:manifold-ext:2020.1.24'
+    implementation 'systems.manifold:manifold-ext-rt:2020.1.24'
```

We need to exclude a runtime dependency to manifold in order to build and run uberjar without it.
Check `./gradlew uberjar && ! jar tf build/libs/karkinos-standalone-2.0.5-SNAPSHOT.jar | grep manifold; java -jar build/libs/karkinos-standalone-2.0.5-SNAPSHOT.jar analysis`

```suggestion
    testImplementation 'systems.manifold:manifold-ext-rt:2020.1.24'
```